indigo_api/static/stylesheets/_export_base.scss

```diff
@@ -18,7 +18,7 @@ body.pdf {
   .akn-chapter {
     page-break-inside: avoid;
   }
-
+  la-akoma-ntoso,
```

Comment:
I consider this a custom style indigo-web

Reply:
This file can remain unchanged. It's used to build up the stylesheets included in ePUB and PDF. For both of those formats, we currently use `<div class="akoma-ntoso">` because neither format supports custom elements (as far as I know).

Reply:
cool reverted the changes